alma/credentials/alma_session.py

```diff
@@ -0,0 +1,10 @@
+from . import Credentials
+
+
+class AlmaSessionCredentials(Credentials):
+    def __init__(self, session_id: str, cookie_name: str):
+        self.session_id = session_id
+        self.cookie_name = cookie_name
+
+    def configure(self, request):
+        request.cookies = {**request.cookies, self.cookie_name: self.session_id}
```

Comment:
I think these objects are mutable and while in JavaScript it would be a good practice to clone them, in Python it is better not to clone them so that objects using the reference keep your updates.

```suggestion
        request.cookies[self.cookie_name] = self.session_id
```